libkbfs/count_meter.go

```diff
@@ -0,0 +1,172 @@
+package libkbfs
+
+import (
+	"sync"
+	"time"
+
+	metrics "github.com/rcrowley/go-metrics"
+)
+
+// MeterSnapshot is a read-only copy of another Meter.
+type MeterSnapshot struct {
```

Comment:
Please add this line somewhere:

```go
var _ metrics.Meter = (*MeterSnapshot)(nil)
```

Also, if this is copied from go-metrics, please say so, and say why it's copied (presumably because you can't instantiate one of those from another package?).